test/functional/apps/discover/_runtime_fields_editor.ts

```diff
@@ -79,6 +78,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await fieldEditor.save();
       await fieldEditor.confirmSave();
       await PageObjects.header.waitUntilLoadingHasFinished();
+      await PageObjects.discover.waitForDocTableLoadingComplete();
```

Comment:
Flaky ones are sometimes a mystery, I wonder why this resolved the issue, I would have expected the retry on L83 to prevent flakiness. Just to be sure, it wouldn't fail on the 51the try, I started a 100 runs flaky test. LGTM https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/666